packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2202,9 +2202,17 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 /// without making the child available for hit testing, and without taking any
 /// room in the parent.
 ///
+/// Animations continue to run in offstage children and may cause unintentional
+/// battery drain.
+///
+/// Offstage is usually used to measure the dimensions of a widget without
+/// bringing it on screen (yet). It should not be used to (temporarily)
```

Comment:
Rather than the last sentence here, consider something like `To remove a widget from view while it is not needed, prefer  removing the widget from the tree entirely rather than keeping it alive in an [Offstage] subtree.`